app/src/main/java/com/example/tasky/agenda/domain/model/Agenda.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.tasky.agenda.domain.model
+
+sealed class Agenda {
```

Comment:
Which is the benefit to add those parameters to AgendaItem and override them? Since I will never use plain agenda object, I think in my case that just add innecesary code

Reply:
The benefit is that you can access them without needing to cast:
```
val agendaItems = listOf<AgendaItem>(Event(...), Task(...), Reminder(...))

agendaItems.forEach { item ->
    println(item.id) // Doesn't work because AgendaItem has no id field
}
```

Reply:
Yes, but I dont think I will use it that way in the app